tests/resource/search.no-auth.spec.ts

```diff
@@ -0,0 +1,77 @@
+import {
+    CONTENT_TYPE,
+    TESTNET_RESOURCE,
+    TESTNET_DID_IDENTIFIER,
+    TESTNET_DID_RESOURCE_ID,
+    DID_METHOD, TESTNET_DID,
+    TESTNET_DID_CREATED_TIME,
+    NOT_EXISTENT_TESTNET_DID,
+    NOT_EXISTENT_RESOURCE_ID,
+    NOT_EXISTENT_TESTNET_DID_IDENTIFIER
+} from '../constants';
+import { test, expect } from '@playwright/test';
+import { StatusCodes } from 'http-status-codes';
+
+test('[Positive] It can search resource with an existent DID and resourceId', async ({ request }) => {
+    const response = await request.get(`/resource/search/${TESTNET_DID}?resourceId=${TESTNET_DID_RESOURCE_ID}`);
+    expect(response.status()).toBe(StatusCodes.OK);
+
+    const body = (await response.body()).toString();
+
+    expect(body).toEqual(TESTNET_RESOURCE);
```

Comment:
Test for a known image or JSON instead, that way you can also check whether MIME headers are passed correctly.